libnmstate/nm/sriov.py

```diff
@@ -162,42 +148,66 @@ def get_info(device):
     except FileNotFoundError:
         return info
 
-    connection = nm_connection.ConnectionProfile()
-    connection.import_by_device(device)
-    if not connection.profile:
-        info[Ethernet.SRIOV_SUBTREE] = sriov_config
-        return info
+    if sriov_config[Ethernet.SRIOV.TOTAL_VFS]:
```

Comment:
`sriov_config` is no longer a "config", it is now a sriov running_info or something similar.

Reply:
Done!